sync/src/synchronizer/mod.rs

```diff
@@ -260,12 +260,17 @@ impl Synchronizer {
         message: packed::SyncMessageUnionReader<'r>,
     ) {
         let item_name = message.item_name();
+        let item_bytes = message.as_slice().len() as u64;
         let status = self.try_process(nc, peer, message);
 
-        metrics!(counter, "ckb-net.received", 1, "action" => "sync", "item" => item_name.to_owned());
-        if !status.is_ok() {
-            metrics!(counter, "ckb-net.status", 1, "action" => "sync", "status" => status.tag());
-        }
+        metrics!(
```

Comment:
I think this metrics should be moved to https://github.com/nervosnetwork/ckb/blob/a8829f4a54ec453c48b67808c5ec694a09cfae2a/sync/src/relayer/mod.rs#L660

and use the same metrics parameters as direction out
```
    metrics!(
        counter,
        "ckb.messages_bytes", msg.,
        "direction" => "in",
        "protocol_id" => SupportProtocols::Relay::protocol_id(),
        "item_id" => msg.item_id(),
    );
```